awx_collection/plugins/modules/user.py

```diff
@@ -163,6 +163,10 @@ def main():
     if password is not None:
         new_fields['password'] = password
 
+    # Prevent password reset when the user exists and update_secrets is False
+    if existing_item and existing_item.get('password') is None and not module.update_secrets:
+      del new_fields['password']
```

Comment:
looks like indentation is off from prior lines in this file. I'd like to have that consistent.